product-service/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
```

Comment:
Import syntax may be used.

Reply:
With utility files and webpack configs i prefer use `require` for make sure that it will be run in all environment